### DISCUSSION TITLE
Looking for studying partners

### DISCUSSION THREAD

**Discussion by JohnEMagee** [smartcontractkit/full-blockchain-solidity-course-py#1207] [General]
WOndering if anyone out there is looking for study/working partners.  I'm about 3.5 hours in, on a mac osx system (you can see my messages regarding struggles that have kept me from moving forward the past couple days) but always find I work better with motivation.  

Would also be interesting in hooking up with folk who might want to work on a project long term if goals sync up (not sure if this is appropriate for here but I'm sure I'm not the only one doing this alone)

**Comment by cromewar** [comment]
Hello @JohnEMagee I think the best place to find a partner (in crime) is to look into the chainlink discord sever:
https://discord.com/invite/aSK4zew

**Comment by n4n0b1t3** [comment]
Hi @JohnEMagee, yes, I feel the same way. I'm inching through the course in small steps (fits my name) but it's painfully slow. Currently, I'm in lesson 7 and some concepts take some getting used to. It helps to have someone to talk it through with. But you need someone who is at the same place in the course and has the same pace. Over the last few weeks, I've written a few articles on certain aspects of the course, which has helped my understanding but slowed me down even more. So what @cromewar says is a good idea, try to find someone on Discord who is at the same level as you. If it doesn't work out, find someone else.

**Comment by Noveleader** [comment]
> WOndering if anyone out there is looking for study/working partners. I'm about 3.5 hours in, on a mac osx system (you can see my messages regarding struggles that have kept me from moving forward the past couple days) but always find I work better with motivation.
> 
> Would also be interesting in hooking up with folk who might want to work on a project long term if goals sync up (not sure if this is appropriate for here but I'm sure I'm not the only one doing this alone)

Hey @JohnEMagee ping me on discord please I am also at about the same length: 
Short Tempered Eye#3765
